backend/dal/events.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+const Promise = require('promise');
+const sqlBuilder = require('../services/sqlBuilder.js');
+
+module.exports = function(connection) {
+  return {
+    types: () => {
```

Comment:
Please use this format
types: {
  'CONVERSATION_IS_VIEWED': 'CONVERSATION_IS_VIEWED',
  'VIDEO_IS_WATCHED': 'VIDEO_IS_WATCHED',
  'VIDEO_IS_WATCHING': 'VIDEO_IS_WATCHING',
  'FILE_IS_DOWNLOADED': 'FILE_IS_DOWNLOADED'
}